docs/user-guide/sdks-using.md

```diff
@@ -2,22 +2,38 @@
 
 Learn about using the Zowe Client Software Development Kits (SDKs). The SDKs enable you to build client applications and scripts that interface with the mainframe.
 
-The SDKs include programamatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
 
 The following SDKs are available.
-- Zowe Node Client SDK
+- Zowe Node.js Client SDK
 - Zowe Python Client SDK
 
 ## Software requirements
 
-Before you use the Python SDK, install the following software:
+### Node.js
 
-<!-- These are listed in the python repo. I assume that the end user needs to install them prior to calling the APIs? Do they install them into their project? -->
+If you download Node SDK packages **from Zowe.org**, the downloaded folder contains dependencies that you must install manually. Copy the files to your project and issue the following commands:
```

Comment:
Before mentioning "the downloaded folder", should we tell users to extract the TGZ that they've downloaded?